lib/ws/ws.js

```diff
@@ -485,7 +492,8 @@ BeamSocket.prototype.close = function () {
  * we try to require Bluebird, but you can switch it out with your own.
  * @type {Promise}
  */
-try { BeamSocket.Promise = require('bluebird'); } catch (e) {}
+// eslint-disable-next-line global-require
```

Comment:
In previous cases I saw requires in runtime code, which were obviously slowing things down which is the reason I enabled this rule.
Sadly this has a few _false positives_ (not really false since the rule never promised to solve this problem).
Would you rather have me disable this rule?